public/styles/icons.less

```diff
@@ -17,7 +17,6 @@
   vertical-align: middle;
   display: inline-block;
   text-decoration: inherit;
-  width: 1em;
```

Comment:
Double-checking, was this on purpose?

Reply:
Yes, it was causing some inconsistencies on margins around the icons. Now it's all perfect again :)